pkg/master/master.go

```diff
@@ -182,8 +182,10 @@ type Config struct {
 	// allow downstream consumers to disable swagger
 	EnableSwaggerSupport bool
 	// allow api versions to be conditionally disabled
-	DisableV1 bool
-	EnableExp bool
+	DisableV1                 bool
+	EnableExtensions          bool
+	EnableExtensionResources  map[string]struct{} // If present, only these resources are enabled in the extensions api group, all others are disabled. This is ignored if EnableExtensions is false.
```

Comment:
Are you suggesting combining Enable and Disable maps or just changing each of them to map[string]bool?